src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $paddingCalendarBox;
+  width: $calendarSize;
+  margin: auto;
+  padding-top: 82px;
+
+  &__day {
+    font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif;
+    font-size: 30px;
+    text-decoration: none;
+    color: black;
+
+    background-color: #eee;
+    border: 1px solid black;
+
+    height: $heightCell;
+    width: $widthCell;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    transition: all 0.5s;
```

Comment:
list transition properties instead of using `all`, it is better for performance and move 0.5s to a variable